src/app/presentation/overview/overview.component.html

```diff
@@ -0,0 +1,6 @@
+<div class="overview-buttons">
+  <button (click)="toggle(presentation.mode !== 1)">
+      {{presentation.mode === 1 ? 'Back' : 'Overview'}}
+  </button>
+  <button *ngIf="presentation.mode === 1" (click)="print()">Print</button>
```

Comment:
Please refer to the enum rather than the number. You can either call a function on the component `shouldShowPrintButton` OR expose the enum as a public property on the component as such

```typescript
export class OverviewComponent implements OnInit, OnDestroy {
   PresentationMode = PresentationMode;
...
```

Then use this in your template:

```html
<button *ngIf="presentation.mode === PresentationMode.overview" (click)="print()">Print</button>
```